CHANGELOG.asciidoc

```diff
@@ -119,6 +119,7 @@ https://github.com/elastic/beats/compare/v6.5.0...v7.0.0-alpha1[View commits]
 - Rename `offset` to `log.offset`. {pull}8923[8923]
 - Rename `source_ecs` to `source` in the Filebeat Suricata module. {pull}8983[8983]
 - Remove warnings for deprecated options: "spool_size", "publish_async", "idle_timeout". {pull}9002[9002]
+- Rename many `nginx.access.*` fields to map to ECS. {pull}9081[9081]
```

Comment:
Just a note: We should probably have in 7.0 a doc page generated for all the breaking field changes that users can go to to look up all the fields.

Reply:
Could we automate this via the ecs-migration.yml file?